index.js

```diff
@@ -10,8 +10,12 @@ const {
 
 const { name: moduleName, version: moduleVersion } = require('./package.json')
 
-function defaultFormatSpanName (serviceName, rawReq) {
-  return `${serviceName} - ${rawReq.method} - ${rawReq.url}`
+function defaultFormatSpanName (request) {
+  const parts = [request.method]
+  if(request.routerPath) {
+    parts.push(request.routerPath)
+  }
+  return parts.join(" ")
```

Comment:
Could we simplify this into a ternary?

```suggestion
const { method, routerPath } = request
return routerPath ? `${method} ${routerPath}` : method
```

Reply:
Fixed :)